apps/src/tutorialExplorer/tutorial.jsx

```diff
@@ -0,0 +1,79 @@
+/* A single tutorial shown in TutorialExplorer.  Displays a thumbnail image, name, and some extra information.
+ */
+
+import React from 'react';
+import FilterSet from './filterSet';
+import TutorialDetail from './tutorialDetail';
+import shapes from './shapes';
+import getTagString from './util';
+
+const styles = {
+  tutorialName: {
+    fontFamily: '"Gotham 5r", sans-serif',
+    fontSize: 15,
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden'
+  },
+  tutorialSub: {
+    fontFamily: '"Gotham 3r", sans-serif',
+    fontSize: 12,
+    paddingBottom: 20
+  }
+};
+
+const Tutorial = React.createClass({
+  propTypes: {
+    item: shapes.tutorial.isRequired
+  },
+
+  getInitialState: function () {
```

Comment:
these could just be `getInitialState()` instead of `getInitialState: function() {`